scripts/remove_scsi.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import os
+
+
+def main(work_directory):
+    list_of_files = list()
+    for (dir_path, dir_names, file_names) in os.walk(work_directory):
+        list_of_files += [os.path.join(dir_path, file) for file in file_names]
+    for file in list_of_files:
+        if file.endswith(".scs"):
+            is_file_processed = False
+            buffer_file_list = []
+            with open(file, 'r', encoding='utf-8') as scs_file:
+                dir_path = os.path.split(file)[0]
+                for scs_file_line in scs_file:
+                    if scs_file_line.find('[*^"file://') >= 0:
+                        edit_line_list = scs_file_line.split('^"file://')
+                        tabulation_string = ""
+                        for _ in edit_line_list[0]:
+                            tabulation_string = tabulation_string + " "
+
+                        buffer_file_list.append(edit_line_list[0] + '\n')
+                        scsi_path = edit_line_list[1].split('"*];;')[0]
+
+                        relative_path = ""
+                        if scsi_path.find('/') >= 0:
+                            scsi_path_list = scsi_path.split('/')
+                            for i in range(len(scsi_path_list) - 1):
+                                relative_path = scsi_path_list[i] + '/'

```

Comment:
Here it should be `relative_path += scsi_path_list[i] + '/'`, but there is even an easier way to write it
```suggestion
                        if scsi_path.rfind('/') >= 0:
                            relative_path = scsi_path[:scsi_path.rfind('/') + 1]
```